### PR TITLE
Exchange orders

### PR DESCRIPTION
**Pull request by Affan007**

### CODE REVIEW

File: exchange-orders/handler.js [hunk 1]
```diff
@@ -14,24 +14,48 @@ exports.exchange = async (event, context, callback) => {
...
-            "properties": [
-              {
-                "name": "size",
-                "value": body.size
-              },
-            ],
-            "price": body.amount,
-            "Discount":body.discount,
-            
+            "code": "bbbb",
```
Comment (PygmalionPolymorph) [inline exchange-orders/handler.js]: bbbb?

File: braintree-pay/serverless.yml [hunk 2]
```diff
@@ -1,3 +1,5 @@
+org: affan
```
Comment (PygmalionPolymorph) [inline braintree-pay/serverless.yml]: Please don't claim code ownership like this

File: braintree-pay/handler.js [hunk 3]
```diff
@@ -108,3 +109,203 @@ exports.clientToken = async event => {
...
+      body: JSON.stringify({ success: false, message: 'Please provide fields with body.' })
+    }
+  }
+  const body = JSON.parse(event.body)
+
+  const { email, totalAmount, customerId, updateActualOrder } = body
+
+  if (!email || !totalAmount || !customerId ) {
+    return {
+      statusCode: 200,
```
[the remark this answers is not in the document]
Comment (Affan007) [reply braintree-pay/handler.js]: it's a graceful failing the api, instaed of bad request , it sends states code false,
there was an other function on top done like that, i thought it's the practice we are following :) 
body: JSON.stringify({ success: false, message: 'Please provide fields with body.' })

File: exchange-orders/handler.js (hunk 1 above)
Comment (Affan007) [reply exchange-orders/handler.js]: not sure either that's the statement that was being added in orders in shopify, that's why i write it 
Screenshot from 2020-02-24 00-10-05

File: braintree-pay/serverless.yml (hunk 2 above)
Comment (Affan007) [reply braintree-pay/serverless.yml]: oh i don't remember adding this,is probably added while deploying using serverless, but not sure, will remove it from here and exchange-order as well

File: exchange-orders/handler.js (hunk 1 above)
Comment (PygmalionPolymorph) [reply exchange-orders/handler.js]: Uhh.. okay
😅

**Pull request merged**